main.py

```diff
@@ -2,6 +2,31 @@
 
 from django.db.models import QuerySet
 
+from db.models import Genre, Actor
+
 
 def main() -> QuerySet:
-    pass
+    genres_list = ["Western", "Action", "Drama"]
+    names_list = [
+        ("George", "Klooney"), ("Kianu", "Reaves"), ("Scarlett", "Keegan"),
+        ("Will", "Smith"), ("Jaden", "Smith"), ("Scarlett", "Johansson")
+    ]
+
+    for genre in genres_list:
+        Genre.objects.create(name=genre)
+
+    for first_name, last_name in names_list:
+        Actor.objects.create(first_name=first_name, last_name=last_name)
+
+    Genre.objects.filter(name="Drama")
+
+    Actor.objects.filter(last_name="Klooney").update(last_name="Clooney")
```

Comment:
It is working, but it's better to add the first name to the filter, according to the task 

![image](https://user-images.githubusercontent.com/104218550/224380466-7805ffc4-4664-4a46-b0d4-6d6000c62015.png)